File: Beginner/1065 - Even Between five Numbers/1065.c
#include <stdio.h>
#include <math.h>

int main(){

    int i, a = 0;
    int A[5];

    for(i=0; i<5; i++){
        scanf("%i", &A[i]);
        if(A[i]%2 == 0)
            a++;
    }
    printf("%i valores pares\n", a);
    return 0;
}

File: Beginner/1065 - Even Between five Numbers/1065.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int i, a = 0;
        int[] A = new int[5];

        for (i = 0; i < 5; i++)
        {
            A[i] = int.Parse(Console.ReadLine());
            if (A[i] % 2 == 0)
                a++;
        }

        Console.WriteLine($"{a} valores pares");
    }
}